INV4/pallet-ips/src/benchmarking.rs

```diff
@@ -0,0 +1,62 @@
+//! Benchmarks for IPS Pallet
+#![cfg(feature = "runtime-benchmarks")]
+
+pub use super::*;
+use frame_benchmarking::{benchmarks, impl_benchmark_test_suite, vec, whitelisted_caller};
+use frame_system::RawOrigin;
+use primitives::{
+    InvArchLicenses,
+    OneOrPercent::{One, ZeroPoint},
+};
+use sp_runtime::Percent;
+
+macro_rules! percent {
+    ($x:expr) => {
+        ZeroPoint(Percent::from_percent($x))
+    };
+}
+
+benchmarks! {
+    where_clause {
+        where T: ipl::Config<Licenses = InvArchLicenses>
+    }
+    create_ips {
+        let caller: T::AccountId = whitelisted_caller();
+        let metadata: Vec<u8> = vec![1];
+        let data: Vec<<T as ipf::Config>::IpfId> = vec![];
+        let license = InvArchLicenses::GPLv3;
+    }: _(RawOrigin::Signed(caller), metadata, data, true, None, license, percent!(50), One, false)
```

Comment:
IPS being created with IPFs that were never minted, same issue as burning without minting

Reply:
Unresolved yet